extensions/amp-story/1.0/amp-story.js

```diff
@@ -451,6 +452,7 @@ export class AmpStory extends AMP.BaseElement {
     const pageIds = this.pages_.map(page => page.element.id);
     this.element.appendChild(this.systemLayer_.build(pageIds));
     this.updateAudioIcon_();
+    this.checkForSidebar_();
```

Comment:
Could you please do this before building the system layer?
That way, we make sure the system layer will be built and appended with the button, ensuring the content doesn't blink if the sidebar icon was to appear later.

You'll have to modify the `this.storeService_.subscribe(...` method in the system layer, to add a third `true /** callToInitialize */` parameter, that will call the event handler immediately with the `hasSidebarState` value.